README.md

```diff
@@ -41,15 +38,17 @@ the Checked C implementation has not broken existing tests.
 Note: These steps have been adopted from the [LNT Quickstart Guide](http://llvm.org/docs/lnt/quickstart.html).
 ```
 sudo apt-get install bison flex
```

Comment:
1. We might want to mention that these instructions are for Ubuntu 20.
2. Ubuntu 20 supports `sudo apt install` instead of `sudo apt-get install`.
3. We can combine multiple installs in a single line like `sudo apt install bison flex tclsh`.
4. Since we use `apt install` to install `virtualenv` we no longer need `python3-pip` and `setuptools`. So we can get rid of the lines: 
```
sudo apt-get install python3-pip
sudo pip3 install setuptools
```

Reply:
Yes. Will make these updates.